manage_user_page.php

```diff
@@ -64,6 +64,8 @@
 $f_filter        = mb_strtoupper( gpc_get_string( 'filter', config_get( 'default_manage_user_prefix' ) ) );
 $f_page_number   = gpc_get_int( 'page_number', 1 );
 
+$f_findname	 = mb_strtoupper( gpc_get_string( 'findname', '' ) ); 
```

Comment:
mb_strtoupper not needed as db_helper_like provides case-insensitive comparison.